cli/commands/getBalance.ts

```diff
@@ -22,7 +22,7 @@ export class GetBalance extends Command<CurrencyContext> {
     const allTokens = { ...builtinTokens, ...customTokens }
     const { currencyCode } = tokenId == null ? currencyInfo : allTokens[tokenId]
 
-    const nativeBalance = await engine.getBalance({ currencyCode })
+    const nativeBalance = await engine.getBalance({ tokenId: tokenId ?? null })
```

Comment:
You can do `const { tokenId = null } = this` up above to get rid of the `?? null` here